### ISSUE TITLE
Merge Weblate translations

### ISSUE THREAD

**Issue by balporig** [Iamlooker/Droid-ify#21]
Fix Weblate merge conflict issues and merger the latest version of all translations. There are several new languages. Also consider Weblate to either automerge changes going forward to main or make it submit MRs with changes.

**Comment by Iamlooker** [comment]
Are they fixed now?

**Comment by balporig** [comment]
I think the moment someone adds a new language or any changes, Weblate complains as it cannot commit them:

```bash
The translation is temporarily closed for contributions due to maintenance, please come back later.
The translation was automatically locked due to following alerts: [Could not push the repository.](https://hosted.weblate.org/projects/droidify/localization/#alerts)
```

Looks like since the split also all new languages and added translations are lost. As NeoStore only added new strings, can we backport those contributions?

**Comment by Iamlooker** [comment]
I believe it is fixed now. Can you check?

**Comment by Iamlooker** [comment]
This has been fixed as seen here 1a0ba2920fdf3735aa637aeff4c3abb34a75cb0a
